MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -41,25 +41,48 @@
 import UIKit
 
 protocol TransactionsTableViewDelegate {
-    func onTransactionSelect(_: Transaction)
+    func onTransactionSelect(_: Any)
```

Comment:
On Android I'm thinking about making all tx types inherit from an abstract Tx class. It would make this line here (and the callee functions) more readable if it was a Tx base type instead of Any. Just an idea.

Reply:
Oh yeah that's why I went with a TransactionProtocol for all the transactions after I started seeing they're all very similar. Would you keep the protocol or go with an abstract class? From my understanding of swift a protocol is usually preffered? 

I removed the `onTransactionSelect ` call for now until I need to go back and work on that ticket.

Reply:
If there's common implementation then I'd go for an abstract base class I think. But I guess you used extensions for that in some classes=